### ISSUE TITLE
[AnalyticalTable]: Accessibility, insufficient contrast of down states using some themes

### ISSUE THREAD

**Issue by ehn-sap** [SAP/ui5-webcomponents-react#5499]
### Describe the bug

While pressing a line of the AnalyticalTable, using some SAP themes, the contrast between text and background is below the required thresholds for accessibility.
Impacted themes: Belize; Quartz Light; Quartz Dark.

### Isolated Example

https://codesandbox.io/p/sandbox/analyticaltable-tree-down-states-colors-t6wgqp

### Reproduction steps

1. Click down a line of the analytical table
2. While the line is pressed, its aspect shows a low contrast in Belize, Quartz Light, or Quartz Dark themes
3. Once clicked up, the aspect and contrast come back to normal or selected, and are ok
...

### Expected Behaviour

The contrast between text and background should remain above the accessibility thresholds

### Screenshots or Videos

![2024-02-14_Belize-down-state_latest](https://github.com/SAP/ui5-webcomponents-react/assets/112404107/b31ac08c-bbc1-4d64-8bde-026b4b0afa68)

### UI5 Web Components for React Version

latest, and previous

### UI5 Web Components Version

latest

### Browser

Chrome

### Declaration

- [X] I’m not disclosing any internal or sensitive information.
